users/models.py

```diff
@@ -136,8 +139,10 @@ def is_admin(self) -> bool:
 @receiver(post_save, sender=CustomUser)
 def update_from_manager_to_employee(sender: "CustomUser", **kwargs: Any) -> None:
     user = kwargs["instance"]
+    logger.info(f"Update user: {user} from manager to employee")
```

Comment:
We should use `{user.pk}` because if you use `{user}` it will print what is defined in `__str__` method in `CustomUser`. And currently there is nothing defined there, and event if there is it might change in the future. Using `pk` guarantees that we are always able to identify the user in database if it still exists. Please apply to other logs too.

Reply:
Done